### PR TITLE
Make headings consistent

### PR DESCRIPTION
**Pull request by finn0**
## Thanks for contributing a pull request!

If you are contributing for the first time or provide a trivial fix don't worry too
much about the checklist - we will help you get started.

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [ ] wrote descriptive pull request text
- [ ] added/updated test(s)
- [ ] updated/extended the documentation
- [ ] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/)
      in message body
- [ ] added news fragment in [changelog folder](../tree/master/docs/changelog)
  * fragment name: `<issue number>.<type>.rst` for example (588.bugfix.rst)
  * `<type>` is must be one of `bugfix`, `feature`, `deprecation`,`breaking`, `doc`, `misc`
  * if PR has no issue: consider creating one first or change it to the PR number after creating the PR
  * "sign" fragment with "by :user:`<your username>`"
  * please use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files - by :user:`superuser`."
  * also see [examples](../tree/master/docs/changelog)
- [ ] added yourself to `CONTRIBUTORS` (preserving alphabetical order)

### CODE REVIEW

**Comment by asottile** [comment]
seems most of them are lower case, I'd prefer to go that way instead personally

**Comment by finn0** [comment]
@asottile, then should I make all them of small, instead of caps, to make document style consistent?

File: docs/config.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
 .. be in -*- rst -*- mode!
 
-tox configuration specification
+tox Configuration Specification
```
Comment (gaborbernat) [inline docs/config.rst:3]: I see no reason to change anything on this line

File: docs/example/basic.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
-Basic usage
+Basic Usage
```
Comment (gaborbernat) [inline docs/example/basic.rst:1]: no need to change second word capitalization

File: docs/example/devenv.rst [hunk 3]
```diff
@@ -1,5 +1,5 @@
 =======================
-Development environment
+Development Environment
```
Comment (gaborbernat) [inline docs/example/devenv.rst:2]: no need to change second word capitalization

File: docs/example/documentation.rst [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Generate documentation
+Generate Documentation
```
Comment (gaborbernat) [inline docs/example/documentation.rst:1]: no need to change second word capitalization

File: docs/example/general.rst [hunk 5]
```diff
@@ -1,6 +1,6 @@
 .. be in -*- rst -*- mode!
 
-General tips and tricks
+General Tips and Tricks
```
Comment (gaborbernat) [inline docs/example/general.rst:3]: no need to change second and further word capitalization

File: docs/example/platform.rst [hunk 6]
```diff
@@ -1,4 +1,4 @@
-Platform specification
+Platform Specification
```
Comment (gaborbernat) [inline docs/example/platform.rst:1]: no need to change second word capitalization

File: docs/example/result.rst [hunk 7]
```diff
@@ -1,4 +1,4 @@
-Writing a json result file
+Writing a JSON Result File
```
Comment (gaborbernat) [inline docs/example/result.rst:1]: ``result`` file should be lowercase

File: docs/index.rst [hunk 8]
```diff
@@ -1,7 +1,7 @@
-Welcome to the tox automation project
+Welcome to the tox Automation Project
```
Comment (gaborbernat) [inline docs/index.rst:1]: no need to change second word capitalization

File: docs/install.rst [hunk 9]
```diff
@@ -1,4 +1,4 @@
-tox installation
+tox Installation
```
Comment (gaborbernat) [inline docs/install.rst:1]: no need to change second word capitalization

File: docs/install.rst [hunk 10]
```diff
@@ -49,7 +49,7 @@ or install it `editable <https://pip.pypa.io/en/stable/reference/pip_install/#ed
 so that you can do changes and submit patches.
 
 
-[Linux/macOS] Install via your package manager
+Install via your package manager - Linux/macOS
```
Comment (gaborbernat) [inline docs/install.rst:52]: Please revert this change, we don't want to break the existing hyperlink.

File: docs/plugins.rst [hunk 11]
```diff
@@ -1,6 +1,6 @@
 .. be in -*- rst -*- mode!
 
-tox plugins
+tox Plugins
```
Comment (gaborbernat) [inline docs/plugins.rst:3]: no need to change second word capitalization

File: docs/support.rst [hunk 12]
```diff
@@ -1,19 +1,19 @@
 .. _support:
 
-support and contact channels
+Support and Contact Channels
```
Comment (gaborbernat) [inline docs/support.rst:3]: no need to change second and further word capitalization

**Review by gaborbernat — CHANGES_REQUESTED** [review]
I'm ok with first letter starting with capital per grammar rules, but please no to further words

File: docs/config.rst (hunk 1 above)
Comment (finn0) [reply docs/config.rst:3]: Actually, I tried to make first letter of each word of document title as caps. I've seen many people follows this convention. If you want to revert these changes, I'm more than happy to make suggested changes.

File: docs/install.rst (hunk 10 above)
Comment (finn0) [reply docs/install.rst:52]: If issue only is hyperlink breakage, I'll try to fix all hyperlink breakage.

**Comment by finn0** [comment]
Sorry for cluttering pull request too much, I should have first opened an issue to discuss these things, before making a pull request.

File: docs/install.rst (hunk 10 above)
Comment (gaborbernat) [reply docs/install.rst:52]: No, please revert this line as a whole.

File: docs/config.rst (hunk 1 above)
Comment (gaborbernat) [reply docs/config.rst:3]: Yes, I'd like to revert.

File: docs/examples.rst [hunk 13]
```diff
@@ -1,5 +1,4 @@
-
-tox configuration and usage examples
+tox Configuration and Usage Examples
```
Comment (finn0) [inline docs/examples.rst:1]: @gaborbernat do you want me to make changes, here also? See, second and further words are caps.

Comment (gaborbernat) [reply docs/examples.rst:1]: Yes. I only want capital first word.

**Comment by finn0** [comment]
@gaborbernat, can you take a look into PR, once again? I've made all suggested changes.

**Review by gaborbernat — APPROVED** [review]

**Comment by gaborbernat** [comment]
thank
